src/Security/Authentication/test/JwtBearerTests.cs

```diff
@@ -1017,8 +1017,7 @@ public void CanReadJwtBearerOptionsFromConfig()
         var config = new ConfigurationBuilder().AddInMemoryCollection(new[]
         {
             new KeyValuePair<string, string>("Authentication:Schemes:Bearer:ValidIssuer", "dotnet-user-jwts"),
-            new KeyValuePair<string, string>("Authentication:Schemes:Bearer:ValidAudiences:0", "http://localhost:5000"),
-            new KeyValuePair<string, string>("Authentication:Schemes:Bearer:ValidAudiences:1", "https://localhost:5001"),
+            new KeyValuePair<string, string>("Authentication:Schemes:Bearer:ValidAudience", "http://localhost:5000"),
```

Comment:
Can we include a test that configures both `ValidIssuer` and `ValidIssuers`? Or `ValidAudience` and `ValidAudiences`?